bridgepoint/prebuild.py

```diff
@@ -1930,7 +1984,7 @@ def prebuild_action(instance):
     return walker.accept(root)
     
                 
-def prebuild_model(metamodel):
+def prebuild_model(metamodel, parent_paths=[]):
```

Comment:
Python has a somewhat unexpected behavior with mutable default values: 
```
def append(number, number_list=[]):
    number_list.append(number)
    print(number_list)
    return number_list

append(5) # expecting: [5], actual: [5]
append(7) # expecting: [7], actual: [5, 7]
append(2) # expecting: [2], actual: [5, 7, 2]
```

Perhaps it does not matter in this case, but I usually do it like this instead:

```
def prebuild_model(metamodel, parent_paths=None):
    parent_paths = parent_paths or list()
    ...
```

Or you could use a default tuple() value instead, which is immutable.

Reply:
I think I will change it to have no default since the option parser will handle passing a default value if the user does not provide one.

Reply:
Actually, I will go with your suggestion since maybe that is more expressive when reading the code...